primitives/src/transaction.rs

```diff
@@ -352,8 +352,10 @@ pub struct TxOut {
 #[cfg(feature = "alloc")]
 impl TxOut {
     /// This is used as a "null txout" in consensus signing code.
-    pub const NULL: Self =
-        TxOut { value: Amount::from_sat(0xffffffffffffffff), script_pubkey: ScriptBuf::new() };
+    pub const NULL: Self = TxOut {
+        value: Amount::from_sat_unchecked(0xffffffffffffffff),
```

Comment:
In the superseeded PR I had moved `NULL` to be a const since this value breaks the range invariant. **https://github.com/yancyribbens/rust-bitcoin/blob/267b728ec21a30656584f3f8b724f651992da8ba/units/src/amount/unsigned.rs#L58**.  That could be piecemealed into it's own separate PR.

Reply:
At first glance I don't see how this doesn't panic using from_sat_unchecked().

Reply:
Oh wait, this is the version that doesn't enforce the range invariant which is why this works.  Although, if we consider changing this in the future to enforce it, lets not violate it here if possible.